src/OrchardCore/OrchardCore.ContentManagement.Display/ContentDisplay/ContentFieldDisplayDriver.cs

```diff
@@ -11,8 +11,11 @@ namespace OrchardCore.ContentManagement.Display.ContentDisplay
 {
     public abstract class ContentFieldDisplayDriver<TField> : DisplayDriverBase, IContentFieldDisplayDriver where TField : ContentField, new()
     {
+        private const string _Display = "_Display";
```

Comment:
I dislike the underscore for constants

Reply:
Oh i missed this one, i will rename it, this was because a `Display` method already exists.